tests/unittests/tests-uri_parser/tests-uri_parser.c

```diff
@@ -48,8 +48,7 @@
         }                                                                   \
     } while (0)
 
-#define VEC_MSG_LEN (sizeof("Unexpected userinfo member \"\" for \"\"") + \
-                     64U + 8U)
+#define VEC_MSG_LEN   (256)
```

Comment:
This was like ~110 before. tests are executed pretty low on the stack, so this *should* be ok.

Reply:
Can you tell which test exactly tripped the asan? The original `VEC_MSG_LEN` might be based on a wrong assumption, which in testing is always a warning sign.

Reply:
(so I'd rather fix the value for the correct assumption than just making it large enough, to account for exactly those kind of wrong assumptions)

Reply:
Ah... Just saw, that this is just for the output messages of the assert... so nvm.

Reply:
jup, my debugging went the same way, but it was actually the first test (so first message) where it tripped. Then I tried to parse the macro for why this fails, but that was not very fruitful.